bundles/org.openhab.binding.monopriceaudio/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,545 @@
+				<label>Zone 13</label>
+				<description>The Controls for Zone 13</description>
+			</channel-group>
+			<channel-group id="zone14" typeId="zone14">
+				<label>Zone 14</label>
+				<description>The Controls for Zone 14</description>
+			</channel-group>
+			<channel-group id="zone15" typeId="zone15">
+				<label>Zone 15</label>
+				<description>The Controls for Zone 15</description>
+			</channel-group>
+			<channel-group id="zone16" typeId="zone16">
+				<label>Zone 16</label>
+				<description>The Controls for Zone 16</description>
+			</channel-group>
+			<channel-group id="zone17" typeId="zone17">
+				<label>Zone 17</label>
+				<description>The Controls for Zone 17</description>
+			</channel-group>
+			<channel-group id="zone18" typeId="zone18">
+				<label>Zone 18</label>
+				<description>The Controls for Zone 18</description>
+			</channel-group>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="false">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>Serial Port to Use for Connecting to the Monoprice Amplifier</description>
+			</parameter>
+			<parameter name="host" type="text" required="false">
+				<context>network-address</context>
+				<label>Address</label>
+				<description>Host Name or IP Address of the Machine Connected to the Monoprice Amplifier (Serial over IP)</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="false">
+				<label>Port</label>
+				<description>Communication Port (IP or Serial over IP). For IP connection to the Monoprice Amplifier</description>
+				<default>4444</default>
+			</parameter>
+			<parameter name="numZones" type="integer" min="1" max="18" required="true">
+				<label>Number of Zones</label>
+				<description>Number of Zones on the Amplifier to Utilize in the Binding (Up to 18 Zones With 3 Amplifiers Connected
+					Together)</description>
+				<default>6</default>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" min="5" max="60" unit="s" required="false">
+				<label>Polling Interval</label>
+				<description>Configures How Often to Poll the Controller to Check for Zone Updates (5-60; Default 15)</description>
+				<default>15</default>
+			</parameter>
+			<parameter name="ignoreZones" type="text" required="false">
+				<label>Ignore Zones</label>
+				<description>(Optional) A Comma Seperated List of Zone Numbers That Will Ignore the 'All Zone' (Except All Off)
+					Commands (ie: 1,6,10)</description>
+			</parameter>
+			<parameter name="initialAllVolume" type="integer" min="1" max="30" required="false">
+				<label>Initial All Volume</label>
+				<description>When 'All' Zones Are Activated, the Volume Will Reset to This Value (1-30; default 10) to Prevent
+					Excessive Blaring of Sound ;)</description>
+				<default>10</default>
+			</parameter>
+			<parameter name="inputLabel1" type="text" required="false">
```

Comment:
Hmm. still not the cleanest solution. It would have been nice if sitemapes would have a feature to use a map properties file, like on item tranformations. So I have no better idea, other than different idea's that might not necessarily be more user friendly. So maybe just leave it as is.